File: executable/node/error-bar-asymmetric-constant.js
var plotly = require('plotly')('test-runner', '9h29fe3l0x')

var data = [
  {
    x: [1, 2, 3, 4], 
    y: [2, 1, 3, 4], 
    error_y: {
      value: 15, 
      type: "percent", 
      symmetric: false, 
      valueminus: 25
    }, 
    type: "scatter"
  }
];

plot_url = plolty.plot(data, function (err, msg) {
    console.log(msg);
});

File: executable/r/error-bar-asymmetric-constant.r
library(plotly)

p <- plotly(username='test-runner', key='9h29fe3l0x')

data <- list(
  list(
    x = c(1, 2, 3, 4), 
    y = c(2, 1, 3, 4), 
    error_y = list(
      value = 15, 
      type = "percent", 
      symmetric = FALSE, 
      valueminus = 25
    ), 
    type = "scatter"
  )
)

response <- p$plotly(data, kwargs=list(filename="error-bar-asymmetric-constant", fileopt="overwrite"))
url <- response$url
filename <- response$filename